libkodimote/libkodimote.pro

```diff
@@ -133,3 +133,8 @@ HEADERS += libkodimote_global.h \
            addonsource.h \
            profiles.h \
            profileitem.h
+
+maemo5 {
+    target.path = /opt/libkodimote/lib
+    INSTALLS += target
+}
```

Comment:
Right... I remember about having had to cancel that warning too all the time with the Nokia SDK. Never bothered to try finding a workaround for it.

CONFIG += static and staticlib seems to be the same according to http://qt-project.org/doc/qt-4.8/qmake-variable-reference.html